README.md

```diff
@@ -79,9 +79,9 @@ To maintain a pristine source tree, CMake encourages to perform an out-of-source
 
     $ mkdir build && cd build
     $ cmake ..
-    $ make
-    $ make check  # run the test suite
-    $ sudo make install  # optional
+    $ cmake --build .
+    $ ctest  # run the test suite
+    $ sudo cmake --build . --target install  # optional
```

Comment:
Is it worth mentioning a modern and more effective alternative, which is available for CMake >= 3.15:
```
$ sudo cmake --install .  # optional
```
?

Reply:
Thanks for the suggestion. In fact, I had this first, but I changed to the current form because we still support CMake 3.13.

Reply:
I mean, mention it additionally, not instead of the current line.

Reply:
However, it's OK in its current view.